docs/reference.md

```diff
@@ -56,15 +56,75 @@ Plugins should response back through browser postMessage mechanism. Each message
 - key - a value from `k` query string parameter. Should be passed as-is
 - data - contain custom embed data or content height of the iframe
 
-Ready message
+## Ready message
 
 Ready message should be send by a plugins as soon as it renders their content and content height is known.
 
     {"source":"custom_embed","action":"ready","key":"j0a","data":{"height":908}}
 
 If ready message will not be sent within ~10 seconds, Ellipsis will render a timeout error and plugin content will be discarded.
 
-Data message
+### Request additional Story ANS
+
+Customers can obtain addition ANS data from Ellipsis via `postMessage`. It can be done with **Search**, **View**, and **Edit** APIs. Ellipsis/Composer will send back the data with the additional fields of:
```

Comment:
u forgot to replace the `APIs` with integrations to match the term we use in this doc at other places

Reply:
opps did not realize what you mean until now, rephrasing them now.